### ISSUE TITLE
Scheduler schedules training twice

### ISSUE THREAD

**Issue by jcklie** [inception-project/inception#1276]
**Describe the bug**
It looks like sometimes, training is running twice.

**To Reproduce**
1. Create a recommender
2. Annotate stuff and look at the logs
3. Sometimes, it loggs training twice

**Expected behavior**
It trains only once

**Please complete the following information:**
 - Version and build ID: 0.10.x

**Comment by reckart** [comment]
There seems to be a bug in `AnnotationDetailEditorPanel` (WebAnno) that `internalCommitAnnotation` and `internalCompleteAnnotation` both call `page.writeEditorCas(aCas)` (which generates the `AfterAnnotationUpdateEvent` to which the recommender service listens for triggering a training) - but only `internalCompleteAnnotation` should be calling that.

**Comment by reckart** [comment]
Fixed upstream issue: https://github.com/webanno/webanno/issues/1346

I wonder if the INCEpTION schedule should be made robust to events being fired in too rapidly... e.g. by allowing to drop new training scheduling if a currently running training task has not been running for longer than 100ms (configurable) - and log a warning in this case.

**Comment by jcklie** [comment]
We could do that by saving the start date in the task and then do the debounce in https://github.com/inception-project/inception/blob/91b4a6c338551586c65db33105ffab22c092ae51/inception-scheduling/src/main/java/de/tudarmstadt/ukp/inception/scheduling/SchedulingService.java#L90 , that should not be so difficult

**Comment by reckart** [comment]
We'd need a `getScheduledTasksAndRunningTasksSince(...)` or something like that.

**Comment by reckart** [comment]
I'm closing this one now since the main issue was fixed upstream. @jcklie if you think we should introduce the robustness, please open a new issue for that.
